### ISSUE TITLE
wrong variable name (likely) in _dinf_flow_distance

### ISSUE THREAD

**Issue by itati01** [mdbartos/pysheds#68]
In the function `_dinf_flow_distance`, Spyder warns of an 'undefined name'. The 3rd line in this fragment should probably changed to `weights_1 = weights[1].ravel()`. Unfortunately, I can currently not verify the code change.

```
weights_0 = weights[0].ravel()
assert(isinstance(weights[1], np.ndarray))
weights_0 = weights[1].ravel() # weights_1?!
assert(weights_0.size == startnodes.size)
assert(weights_1.size == startnodes.size) # Spyder warning: 'undefined name'
```

**Comment by mdbartos** [comment]
Hi @itati01,

You are correct. I think this was never caught in testing because I always specified the weights as an array instead of a list or tuple. Feel free to submit a pull request if you'd like to fix this bug. Otherwise, I will fix as soon as possible.

**Comment by mdbartos** [comment]
Resolved in #73
